composer.json

```diff
@@ -42,7 +42,7 @@
         "phpstan/phpstan-phpunit": "^1.0",
         "phpunit/phpunit": "^9.5",
         "psalm/plugin-phpunit": "^0.17",
-        "psalm/plugin-symfony": "^3.1",
+        "psalm/plugin-symfony": "^4.0.1",
```

Comment:
```suggestion
        "psalm/plugin-symfony": "^4.0",
```

Reply:
The fix is not present on psalm/plugin-symfony v4.0.0 that’s why I required `^4.0.1`.